central-service/main.go

```diff
@@ -1,7 +1,37 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+
+	"github.com/acmutd/bsg/central-service/controllers"
+	"github.com/acmutd/bsg/central-service/models"
+	"github.com/acmutd/bsg/central-service/services"
+)
 
 func main() {
-	fmt.Println("Hello World! Welcome to the Central Service")
+	dsn := "host=db user=bsgdev password=bsgdev dbname=bsg port=5432 sslmode=disable Timezone=America/Chicago"
```

Comment:
Will need to find a way to secure this information.

We can use the Google Secrets Manger package to pull this sensitive information.